en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+        int populationMillions = (populationByCountry[0]?.value ?: 0) / 1000000;
+        decimal totalCasesPerMillion = <decimal>(totalCases / populationMillions);
+        json payload = {country: country, totalCasesPerMillion: totalCasesPerMillion};
+        return payload;
+    }
+}
+```
+Now you can run the REST API and test it to see whether it works as expected.
+
+### Run and test the REST API
+
+To run the API, click **Run** (above the low-code diagram).
+
+Once the REST API is successfully started, the following appears in the terminal log.
+
+```
+Running executable
+```
+Now you can try out the REST API as follows:
+
+1. Click **Try it** As a result, a test view opens to the right of the page.
+
+2. In the test view, expand the **GET** resource.
+
+3. Click **Try it out**.
+
+4. In the **country** field, enter `USA`.
+
+5. Click **Execute**.
+
+The following is displayed as the response body under **Responses**.
+
+![Try-out response](../assets/img/tutorials/rest-api/try-out-response.png){.cInlineImage-half}
+
+The REST API you created works as expected. Therefore, now you can commit it.
+
+### Commit the REST API to GitHub
+
+The REST API you designed is currently available only in the VS Code Editor. To use it, you need to save it in the Choreo Console. You can do this by committing the REST API configuration into a private repository in GitHub as follows:
+
+1. Click **Sync with Choreo Upstream** (at the bottom of the page), and then click **Sync my changes with Choreo**.
+
+2. In the left panel, enter `Implement REST API` as the message and click on the tick.
+
+    ![Commit message](../assets/img/tutorials/rest-api/commit-message.png){.cInlineImage-half} 
+
+    Select **Yes** in the message that appears to specify that you need the changes to be staged.
+
+Once the changes are successfully deployed, the VS Code Editor indicates by displaying the text **In sync with Choreo upstream** for the `service.bal` file.
+
+
+## Deploy
+
+To deploy the API, click the **Deploy** icon. Then in the **Build Area** card, click **Deploy**.
+
+![Deploy API](../assets/img/tutorials/rest-api/deploy-api.png){.cInlineImage-full}
+
+To check the progress of the deployment, click **View Logs**
+
+![Deployment progress](../assets/img/tutorials/rest-api/deployment-progress.png){.cInlineImage-full}
+
+Once the API is deployed, the **Development** card indicates that the API is running as shown below.
+
+![Deployed API](../assets/img/tutorials/rest-api/deployed-api.png){.cInlineImage-full}
+
+Now you can test your deployed REST API to check again whether it is working as expected.
+
+## Test
+
+Once you have deployed the Choreo application you can test it via the Open API Console, A cURL command, or Postman.
+
+For this scenario, let's test via the Open API Console:
+
+1. To open the test view, click **Test** in the **Development** card in the **Deploy tab, or click on the **Test** tab.
```

Comment:
This is kind of confusing, I am not sure if it is because of the bold text, or having two steps in one. We can mention the second as an alternative.